lib/matplotlib/axes/_axes.py

```diff
@@ -7501,6 +7501,9 @@ def specgram(self, x, NFFT=None, Fs=None, Fc=None, detrend=None,
         -----
         The parameters *detrend* and *scale_by_freq* do only apply when *mode*
         is set to 'psd'.
+
+        The origin of the plot is forced to 'upper'. The image.origin rcParam
+        is ignored. Passing origin as a keyword argument raises a `TypeError.`
```

Comment:
```suggestion
        is ignored. Passing origin as a keyword argument raises a `TypeError`.
```

Should also fix the doc build failure (`TypeError.` is no known reference target).